core/templates/dev/head/domain/skill/EditableSkillBackendApiService.ts

```diff
@@ -77,34 +95,10 @@ oppia.factory('EditableSkillBackendApiService', [
       });
     };
 
-    var _fetchQuestions = function(
```

Comment:
Thanks for moving this to the questionbackend :)

Reply:
No problem!